src/app/app.component.ts

```diff
@@ -669,4 +684,14 @@ export class AppComponent implements OnInit {
     this.environmentsService.setActiveView('ENV_LOGS');
     this.environmentsService.setActiveEnvironmentLogTab('RESPONSE');
   }
+
+  /**
+   * Duplicate the active route response
+   */
+  public duplicateRouteResponse() {
+    this.store.selectActiveRouteResponse().pipe(first())
```

Comment:
You can remove this subscription and directly use `this.store.getActiveEnvironment();` which is not an observable. Also, you can put this code in the environment service instead of the app component like there: https://github.com/mockoon/mockoon/blob/master/src/app/services/environments.service.ts#L138

Reply:
I've moved the logic into EnvironmentService#duplicateRouteResponse.